textfinding_indikator10.py

```diff
@@ -0,0 +1,137 @@
+# this code is for TEXT FINDING - INDIKATOR 10
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            reg = f'(?:(tim)\s+(koordinasi)\s+(spbe)?)'
+            if (re.search(reg, line, re.IGNORECASE)):
+                result.append([idx, line])
+
+        elif (lv == 2):
+            for key in list(keyword):
+                if key in keyword:
+                    if re.search(key, line, re.IGNORECASE):
+
+                        # include words
+                        reg = f'(?:(tim)\s+(koordinasi)\s+(spbe)?)'
+                        if re.search(reg, line, re.IGNORECASE):
+
+                            # exclude words
+                            if (not excludewords(line)):
+                                result.append([idx, line])
+                                keyword.remove(key)
+
+        elif (lv == 4):
+            reg = f'(?:(tim)\s+(koordinasi))'
+            if re.search(reg, line, re.IGNORECASE):
+                res = [ele for ele in keyword if (ele in line)]
+                if (res):
+                    if (not excludewords(line)):
+                        result.append([idx, line])
+
+        else:  # lvl 5
+            res = [ele for ele in keyword if (ele in line)]
+            if (res):
+                reg = f'(?:(tim)\s+(koordinasi)\s+(spbe)?)'
+                if re.search(reg, line, re.IGNORECASE):
+                    if (not excludewords(line)):
+                        result.append([idx, line])
+
+        idx += 1
+
+    file.close()
+    return (result)
+
+
+# if __name__ == '__main__':
+#     pdfparser(sys.argv[1])
+
+def excludewords(line):
```

Comment:
Same as [#5 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/5#discussion_r1142379894)

Reply:
Thanks for the feedback! I've added new lines of code. Please give it another review if you've time.